src/natcap/invest/coastal_blue_carbon/coastal_blue_carbon.py

```diff
@@ -1069,18 +1068,17 @@ def _set_up_workspace(args):
         # TypeError when n_workers is None.
         n_workers = -1  # Synchronous mode.
 
-    taskgraph_cache_dir = os.path.join(
-        args['workspace_dir'], TASKGRAPH_CACHE_DIR_NAME)
     task_graph = taskgraph.TaskGraph(
-        taskgraph_cache_dir, n_workers, reporting_interval=5.0)
+        os.path.join(args['workspace_dir'], 'taskgraph_cache'),
+        n_workers, reporting_interval=5.0)
 
     suffix = utils.make_suffix_string(args, 'results_suffix')
     intermediate_dir = os.path.join(
         args['workspace_dir'], INTERMEDIATE_DIR_NAME)
     output_dir = os.path.join(
         args['workspace_dir'], OUTPUT_DIR_NAME)
 
-    utils.make_directories([output_dir, intermediate_dir, taskgraph_cache_dir])
+    utils.make_directories([output_dir, intermediate_dir])
```

Comment:
TaskGraph creates the taskgraph cache dir if it doesn't exist already, right?

Reply:
Correct!